cumulusci/tasks/release_notes/task.py

```diff
@@ -181,3 +182,41 @@ def _get_child_branch_name_from_merge_commit(self):
             )
 
         return child_branch_name
+
+    def _update_unaggregated_pr_header(
+        self, pull_request_to_update, branch_name_to_add
+    ):
+        """Updates the 'Unaggregated Pull Requests' section header with a link
+        to the new child branch pull request"""
+
+        self._add_header(pull_request_to_update)
+
+        pull_requests = get_pull_requests_with_base_branch(
+            self.repo,
+            branch_name_to_add.split("__")[0],
+            branch_name_to_add,
+            state="all",
+        )
+
+        if len(pull_requests) == 0:
+            self.logger.error(f"No pull request for branch {branch_name_to_add} found.")
```

Comment:
```suggestion
            self.logger.info(f"No pull request for branch {branch_name_to_add} found.")
```

I don't think we need to highlight it in red; there are normal situations where this can happen.